### ISSUE TITLE
Adjust our card HTML for accessibility

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/dpul-collections#827]
### Summary or User Story

Our Fable test showed a few issues with cards:

* Titles should be read first (the heading in the card)
* Cards should be in lists so it reads how many there are
* Metadata on cards should be a definition list
* Cards should have genre
* "Images" should be renamed to "Files"

### Acceptance Criteria

- [ ] Titles are read first by a screen reader
- [ ] Cards are in lists (`ol` or `ul`, and they're `li`)
- [ ] Metadata on cards is in a definition list like our metadata pane
- [ ] Cards have the genre.
- [ ] Instead of "Images" it says "Files"

### First step

Start adjusting the card HTML in both the search results and browse item/browse row.
